domains/21z.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "21Z",
+        "email": "",
+        "discord": "780356848737058857"
+    },
+    "record": {
+        "CNAME": "21z.github.io",
+        "TXT": ["_discord.21z.is-a.dev", "dh=cad8d14a40cc3daa14436ff5e874e4c02ce2e1ae"]
```

Comment:
CNAME cant be used for any records (also for discord txt verification you'd need to make another domain aka make _discord.21z.json with the txt discord gave you)